inbm/dispatcher-agent/dispatcher/source/ubuntu_source_manager.py

```diff
@@ -146,26 +146,26 @@ def remove(self, parameters: ApplicationRemoveSourceParameters) -> None:
         # Remove the file under /etc/apt/sources.list.d
         try:
             if (
-                os.path.sep in parameters.file_name
-                or parameters.file_name == ".."
-                or parameters.file_name == "."
+                os.path.sep in parameters.source_list_file_name
+                or parameters.source_list_file_name == ".."
+                or parameters.source_list_file_name == "."
             ):
-                raise SourceError(f"Invalid file name: {parameters.file_name}")
+                raise SourceError(f"Invalid file name: {parameters.source_list_file_name}")
 
             if not remove_file(
                 get_canonical_representation_of_path(
-                    os.path.join(UBUNTU_APT_SOURCES_LIST_D, parameters.file_name)
+                    os.path.join(UBUNTU_APT_SOURCES_LIST_D, parameters.source_list_file_name)
                 )
             ):
-                raise SourceError(f"Error removing file: {parameters.file_name}")
+                raise SourceError(f"Error removing file: {parameters.source_list_file_name}")
         except OSError as e:
             raise SourceError(f"Error removing file: {e}") from e
 
     def update(self, parameters: ApplicationUpdateSourceParameters) -> None:
         """Updates a source file in Ubuntu OS source file list under /etc/apt/sources.list.d"""
         try:
             create_file_with_contents(
-                os.path.join(UBUNTU_APT_SOURCES_LIST_D, parameters.file_name), parameters.sources
+                os.path.join(UBUNTU_APT_SOURCES_LIST_D, parameters.source_list_file_name), parameters.sources
```

Comment:
**Suggestion:** Use a context manager when updating the source list to ensure the file is closed properly in case of an error. [best practice]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
            with open(os.path.join(UBUNTU_APT_SOURCES_LIST_D, parameters.source_list_file_name), 'w') as file:
                for source in parameters.sources:
                    file.write(f"{source}\n")
```